### PR TITLE
DHFPROD-10073: Make test independent

### PR DESCRIPTION
**Pull request by vmendez-marklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran newly added/edited cypress tests on Firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- N/A Added to Release Wiki

### CODE REVIEW

**Review by Sanjeevani19 — COMMENTED** [review]
@vmendez-marklogic Build is failing on the pipeline , converting this to draft for now, please verify it locally and if the build passes you can set the PR to review again

**Comment by vmendez-marklogic** [comment]
For clarity, here is a list of the changes that have been made in general terms:

- All cy.visit() before any login were removed

- The cy.contains(Application.title) was removed for unnecessary scenarios.

- Redundant, unclear or unnecessary comments were removed.

- Spaces and line breaks were modified to have a common standard in all files.

- The postLogin() method was modified by navigateToMainPage(), where instead of an explicit wait, it waits for the necessary requests, and makes it clearer that the method makes you navigate to the main page (avoiding not so clear names). .

- The revertEntitiesChanges command was added to revert changes made to entities in various specs.

- Some particular minor changes so that there are no dependencies with other specs.

**Comment by vmendez-marklogic** [comment]
Failures review:

imagen


writeScenarioGraph.cy.tsx: Failure not related to the changes => 500 server error

imagen

grapchEntitiesSettings.cy.tsx: Failure not related to the changes =>

File: marklogic-data-hub-central/ui/e2e/cypress.config.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ module.exports = defineConfig({
     toConsole: true,
   },
   retries: {
-    runMode: 1,
```
Comment (ngodugu-marklogic) [inline marklogic-data-hub-central/ui/e2e/cypress.config.js:18]: @vmendez-marklogic I have observed few tests failing at first attempt and then going fine in retry and progress further. So may be we can leave the retries as 1 for now. Rest all changes look good to me.

Comment (vmendez-marklogic) [reply marklogic-data-hub-central/ui/e2e/cypress.config.js:18]: Okay! I will revert that change, and we can discuss about retryability, and create a task if it is needed

**Review by aniamy — APPROVED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeNotification.cy.tsx [hunk 2]
```diff
@@ -93,8 +101,11 @@ describe("Merge Notification Functionality From Explore Card View", () => {
       toolbar.closeNotificationModal();
     });
   });
-  it("Run Match and Merge steps to generate Notification Docs", () => {
+
+  it.only("Run Match and Merge steps to generate Notification Docs", () => {
+    cy.intercept("GET", "/api/flows/personJSON/latestJobInfo").as("personJSON");
     cy.waitUntil(() => toolbar.getRunToolbarIcon()).click();
+    cy.wait("@personJSON").wait("@personJSON");
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeNotification.cy.tsx:108]: I suggest changing toogleExpandFlow to toggleFlowAccordion. The accordion is closed so you cannot runStep on line 112.
Also remember to remove the only in the it.

**Comment by aniamy** [comment]
In crudQueriesFromManageQueries the test Verify query selection from All Data view page, doesn't stay on card view is failing because it expected 2 results but got 1. Could it be because of cleaning data?

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeNotification.cy.tsx (hunk 2 above)
Comment (vmendez-marklogic) [reply marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeNotification.cy.tsx:108]: Done!

**Review by aniamy — APPROVED** [review]

**Review by littlecherro — APPROVED** [review]

**Review by ngodugu-marklogic — APPROVED** [review]

**Pull request merged**